.github/workflows/deployment-releases.yml

```diff
@@ -0,0 +1,89 @@
+# Deployment workflow to be used for Jira reporting
+name: Deploy releases
+
+on:
+  release:
+    types: [ published ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  deploy-release:
+    name: Deploy a release
+
+    # IMPORTANT: the workflow must have write access to deployments, otherwise the action will fail.
+    permissions:
+      deployments: write
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match-alpha-release
+        with:
+          text: ${{ github.event.release.name }}
+          regex: 'v\d+\.\d+\.\d+a\d+$'
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match-beta-release
+        with:
+          text: ${{ github.event.release.name }}
+          regex: 'v\d+\.\d+\.\d+b\d+$'
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match-prod-release
+        with:
+          text: ${{ github.event.release.name }}
+          regex: 'v\d+\.\d+\.\d+(.post\d+)?$'
+
+
+      - name: Set release environment
+        id: set_release_env
+        run: |
+          if [ ${{ steps.regex-match-alpha-release.outputs.match }} != "" ]; then
+            echo 'deployment-env=alpha' >> $GITHUB_OUTPUT
+          elif [ ${{ steps.regex-match-beta-release.outputs.match }} != "" ]; then
+            echo 'deployment-env=beta' >> $GITHUB_OUTPUT
+          elif [ ${{ steps.regex-match-prod-release.outputs.match }} != "" ]; then
+            echo 'deployment-env=production' >> $GITHUB_OUTPUT
+          else
+            echo 'deployment-env=unknown' >> $GITHUB_OUTPUT
+            echo 'release name unrecognized: ${{ github.event.release.name }}'
+            exit 1
+          fi
+
+      - uses: chrnorm/deployment-action@v2
+        if: ${{ success() }}
+        name: Create GitHub deployment for the release
+        id: deployment
+        with:
+          token: '${{ github.token }}'
+          environment: ${{ steps.set_release_env.outputs.deployment-env }}
+      # This step could be used in the future to publish release notes or to send Slack notification
```

Comment:
I would suggest the same here as well, to create a ticket and add the id in the comment